json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -507,6 +520,38 @@ public function update_settings() {
 					}
 					break;
 
+				case 'wpcom_publish_posts_with_markdown':
+				case 'wpcom_publish_comments_with_markdown':
+					$coerce_value = (bool) $value;
+					if ( update_option( $key, $coerce_value ) ) {
+						$updated[ $key ] = $coerce_value;
+					}
+					break;
+
+				case 'amp_is_enabled':
+					if ( function_exists( 'wpcom_update_amp_enabled' ) ) {
+						$saved = wpcom_update_amp_enabled( $blog_id, $value );
+						if ( $saved ) {
+							$updated[ $key ] = (bool) $value;
+						}
+					}
+					break;
+
+				case 'site_icon':
```

Comment:
This is a duplicate case and is already handled above (merged previously in #5282). Will need to update the already-included case to account for filter revision from 03b829053497358fd9bbd8265d1dc1514b7016ad.